### ISSUE TITLE
Warning: Failed prop type: Invalid prop `inputContainerStyle` supplied to `Input`.

### ISSUE THREAD

**Issue by freekingg** [react-native-elements/react-native-elements#2765]
```
Warning: Failed prop type: Invalid prop `inputContainerStyle` supplied to `Input`.

```

```
Warning: Invalid DOM property `enterkeyhint`. Did you mean `enterKeyHint`?

```

```
index.js:1 Warning: Failed prop type: Invalid prop `buttonStyle` supplied to `Button`.
```

```
Warning: Failed prop type: Invalid prop `containerStyle` supplied to `Button`.
```

```
"react-native-web": "0.14.0",
"react-native-elements": "^3.1.0",
"react-native": "0.63.4",

```

**Comment by flyingcircle** [comment]
No code? Please provide code and further explanation if you want this ticket to be addressed.

**Comment by freekingg** [comment]
> 没有代码？如果您想处理此票，请提供代码和进一步的说明。

```
import {Button, Input, Icon} from 'react-native-elements';

<Input
            ref={nameInputEl}
            placeholder="手机号/帐号"
            placeholderTextColor="rgba(255,255,255,0.60)"
            leftIcon={{
              type: 'font-awesome',
              name: 'user',
              color: 'rgba(255,255,255,0.60)',
            }}
            inputContainerStyle={styles.inputContainerStyle}
            inputStyle={{
              color: global.colors.base,
            }}
          />



<Button
          title="登录"
          loading={btnLoading}
          loadingProps={{
            color: 'rgba(111, 202, 186, 1)',
          }}
          buttonStyle={styles.btnStyle}
          containerStyle={styles.btnContainerStyle}
          onPress={submitFun}
        />

```

**Comment by flyingcircle** [comment]
I wrote a snack using your example code: https://snack.expo.io/u4w8zM3a2 Something is probably incorrect in your setup, not RNE.

**Comment by khevamann** [comment]
@flyingcircle This is an issue in web-only (React Native Web). Not sure if you fully support RNW but would be very nice to get this warning out of the console.

**Comment by flyingcircle** [comment]
@khevamann , the snack that I presented means that it at least works on expo web. I can reopen this to test it out on web later, but if you want to supply a PR, that would help out the community.

**Comment by khevamann** [comment]
The expo snack is not showing all errors correctly. See https://snack.expo.io/@khevamann/github.com-pavithra-expedux-expo_slider. I passed in notAProp and it is not throwing any warnings or errors. `<Card containerStyle={styles.cardCont} notAProp={22}>` 

The actual error I am getting is below however all other props I pass in throw the same error. It seems to me that the withTheme is messing up the prop types and causing that error.
```sh
Warning: Failed prop type: Invalid prop `containerStyle` supplied to `Card`.
    in Card (at withTheme.js:44)
    in Themed.Card (at MyCardComponent.tsx:26)
```

I don't have the time to look into it more right now, but I will try and get around to it.

**Comment by curiousdustin** [comment]
I am getting a very similar error to @khevamann.

```
Warning: Failed prop type: Invalid prop `containerStyle` supplied to `Input`.
    in Input (at withTheme.js:42)
    in ThemedComponent (at withTheme.js:59)
```

**Comment by hexadecy** [comment]
Yes, it's an issue with React-Native-Web only. 
StyleSheet.create returns **Number** for the style instead of an object or an array.

React-Native-Elements Code:
```jsx
inputContainerStyle?: StyleProp<ViewStyle>;
```

One possible workaround is to pass them in an array:
```jsx
<Input
  inputStyle={[style]}
  inputContainerStyle={[styles.inputContainerStyle]}
  containerStyle={[styles.containerStyle]}
/>
```

I also use this propType definition in my code:
```jsx
PropTypes.oneOfType([PropTypes.object, PropTypes.array, PropTypes.number]);
```

**Comment by flyingcircle** [comment]
PropTypes is now removed from RNE in 3.3.0. Can some tell me if this issue is resolved?

**Comment by hexadecy** [comment]
Yes, it's fixed in RNE 3.3.0!
